docs/my-website/docs/providers/nscale.md

```diff
@@ -0,0 +1,85 @@
+# NScale (EU Sovereign)
+**[NScale docs](https://docs.nscale.com/docs/getting-started/overview)**
+
+NScale is a European-domiciled full-stack AI cloud platform that allows you to scale your workloads securely, sustainably and cost-effectively - without sacrificing control. It provides production-grade reliability for serverless deployments of LLMs with full data sovereignty and compliance baked in.
+
+## Key Features
+- **EU Sovereign**: Full data sovereignty and compliance with European regulations
+- **Ultra-Low Cost (starting at $0.01 / M tokens)**: Extremely competitive pricing for both text and image generation models
+- **Production Grade**: Reliable serverless deployments with full isolation
+- **No Setup Required**: Instant access to compute without infrastructure management
+- **Full Control**: Your data remains private and isolated
+
+## API Key
+```python
```

Comment:
each code sample should have showLineNumbers and title, see https://docs.litellm.ai/docs/providers/meta_llama